packages/cashier/src/pages/on-ramp/on-ramp.scss

```diff
@@ -61,7 +65,7 @@
         }
         &-description {
             grid-area: description;
-            width: 390px;
+            width: 374px;
```

Comment:
can we have this in rem to keep it consistent

Reply:
Ah yes

Reply:
Done.